source/getting-started/user-guide/amds-interface/index.md

```diff
@@ -6,6 +6,15 @@ This document describes the built-in AMDC drivers which can be used to interface
 Before attempting to use these drivers, make sure to read about the AMDS in [its documentation](/accessories/amds/index.md).
 ```
 
+## AMDS Firmware Version
+
+Ensure that your AMDS Firmware version matches your AMDC firmware version using the following table:
+
+| AMDC Version | AMDS Version |
+|-------|-------|
+| v1.3.x or greater | v2.0.x or greater |
+| v1.2.x or lesser  | v1.0.x or greater |
```

Comment:
```suggestion
| v1.2.x or lesser  | v1.x.x |
```

The second row would imply that AMDS `v2.0.0` would work with AMDC `v1.2.x` --- but that's not true. I think you want the AMDS section for this to read `v1.x.x`